code/tpw-hire/tpw-hire.css

```diff
@@ -30,4 +30,112 @@ tr {
   left: 0;
   z-index: 6; /* Increase z-index to be higher priority than the header(lower #) */
   background-color: #f9f9f9;
+  }
+  /*****************/
```

Comment:
Like Susanne mentioned. Just extra spacing before the closing bracket on line 33 and extra spacing before the comment header on line 34. I'd also suggest adding a line in between to create some spacing between the sections.

Reply:
This should also be updated in TPW HiRE | Production CSS. Line 34 spacing is off a bit.